### DISCUSSION TITLE
apisix-etcd版本升级问题

### DISCUSSION THREAD

**Discussion by andylhz-mojie** [apache/apisix#7843] [General]
我在K8S集群中通过helm 部署的apisix ， apisix和dashboard是通过helm upgrade 没问题。但是etcd是有状态的服务，我通过直接更新镜像的方式来升级，但是失败了，那如果我不升级etcd 这个组件会有什么问题吗？

**Comment by tokers** [comment]
@andylhz-mojie Basically you don't have to upgrade ETCD unless there are critical security vulnerabilities in current version.

**Reply by andylhz-mojie** [reply]
Hi tokers, thanks for your reply. Yes ,there are lots of VA  .

<img width="619" alt="WeCom Screenshot_20220902104352" src="https://user-images.githubusercontent.com/18341140/188048178-b179cf2f-2f53-4a13-8474-048e56242c03.png">

**Reply by tokers** [reply]
Try to use `3.5.x`?

**Reply by andylhz-mojie** [reply]
I try to fix the vulnerabilities by rewrite the Docker file and rebuild the image on the old version, all of the vulnerabilities fixed.

FROM docker.io/bitnami/etcd:3.4.16-debian-10-r14
RUN apt-get update -y && apt-get upgrade -y
